tests/UploadedFileTest.php

```diff
@@ -113,7 +115,8 @@ public function invalidFilenamesAndMediaTypes()
      */
     public function testRaisesExceptionOnInvalidClientFilename($filename)
     {
-        $this->expectException('InvalidArgumentException', 'filename');
+        $this->expectException('InvalidArgumentException');
```

Comment:
This should also become a class reference?

Reply:
Done